charts/datadog-crds/values.yaml

```diff
@@ -9,6 +9,12 @@ crds:
   datadogAgents: false
   # crds.datadogMonitors -- Set to true to deploy the DatadogMonitors CRD
   datadogMonitors: false
+  # crds.extendedDaemonSetReplicaSets -- Set to true to deploy the ExtendedDaemonSetReplicaSets CRD
+  extendedDaemonSetReplicaSets: false
+  # crds.extendedDaemonSets -- Set to true to deploy the ExtendedDaemonSet CRD
+  extendedDaemonSets: false
+  # crds.extendedDaemonSetSettings -- Set to true to deploy the ExtendedDaemonSettings CRD
+  extendedDaemonSetSettings: false
```

Comment:
could you remove the `extendedDaemonSetSettings` from this CRD import script, because we don't want to maintain it in the futur.